gramps/gen/proxy/filter.py

```diff
@@ -311,102 +317,102 @@ def get_note_from_handle(self, handle):
         else:
             return None
 
-    def get_person_from_gramps_id(self, val):
+    def get_person_from_gramps_id(self, gramps_id: str) -> Person | None:
```

Comment:
`gramps_id: str`

Is it worth adding a new `GrampsId` type?

Reply:
I should think so. PersonGrampsId, etc... and an AnyGrampsId to align with the added handle types.

Reply:
I've added GrampsId types. @Nick-Hall would you prefer these to be in a separate PR?

Reply:
Yes, please.  Put the type hints in a separate PR from the performance improvements.